setup.py

```diff
@@ -17,7 +17,7 @@
     keywords="microcosm",
     install_requires=[
         "enum34>=1.1.2",
-        "Flask>=0.10.1",
+        "Flask>=0.11",
```

Comment:
I'm not totally sure this is a good idea. I'm also not totally sure that the error handling registration functions worked the same way in `0.10.1`. A cursory read of the source only convinced me that they are *different*.